base_rest/restapi.py

```diff
@@ -131,13 +135,20 @@ def to_response(self, service, result):
         :return: http.Response or JSON dict
         """
 
-    def to_openapi_query_parameters(self, service):
+    @abc.abstractmethod
+    def to_openapi_query_parameters(self, service) -> dict:
         return {}
 
-    def to_openapi_requestbody(self, service):
+    @abc.abstractmethod
+    def to_openapi_requestbody(self, service) -> dict:
         return {}
 
-    def to_openapi_responses(self, service):
+    @abc.abstractmethod
+    def to_openapi_responses(self, service) -> dict:
+        return {}
+
+    @abc.abstractmethod
+    def to_json_schema(self, service, direction) -> dict:
```

Comment:
@lmignon I have a problem with this line.

The function's signature is `to_json_schema(self, service, direction)` in cerberus and datamodel, but in pydantic it is `to_json_schema(self, spec)`.

Since I don't fully understand the base_rest_pydantic module's architecture I don't know what the correct answer to this problem should be

Reply:
I'll check...

Reply:
@PierrickBrun see #268